src/controllers/user.js

```diff
@@ -71,16 +71,38 @@ const User = {
   },
 
   delete: async (req, res, next) => {
-    const { userID } = req.params;
+    const { id } = req.params;
 
     try {
-      await UserModel.findOneAndDelete({ _id: userID });
+      await UserModel.findOneAndDelete({ _id: id });
```

Comment:
Although not necessary, other calls to `findOneAndDelete` are appended with `.exec()`. For consistency, it should be included here.